# Test cases automatically generated by Pynguin (https://www.pynguin.eu).
# Please check them before you use them.
import py_backwards.utils.helpers as module_0
import builtins as module_1
import re as module_2


def test_case_0():
    none_type_0 = None
    none_type_1 = module_0.debug(none_type_0)


def test_case_1():
    variables_generator_0 = module_0.VariablesGenerator()


def test_case_2():
    exception_0 = module_1.Exception()
    none_type_0 = None
    callable_0 = module_0.eager(none_type_0)
    callable_1 = module_0.eager(exception_0)
    variables_generator_0 = module_0.VariablesGenerator()
    str_0 = "tYVya_^E!S"
    none_type_1 = module_0.warn(str_0)
    callable_2 = module_0.eager(callable_1)
    module_0.get_source(callable_1)


def test_case_3():
    var_0 = module_2.purge()
    module_0.get_source(var_0)


def test_case_4():
    none_type_0 = None
    none_type_1 = module_0.warn(none_type_0)


def test_case_5():
    str_0 = "'-<Dn+vnc}U\x0b8*"
    none_type_0 = module_0.warn(str_0)
    none_type_1 = None
    callable_0 = module_0.eager(none_type_1)
    variables_generator_0 = module_0.VariablesGenerator()
    callable_0.__call__(variables_generator_0, none_type_1, callable_0)
